<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="head0">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>8</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>求和</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Build Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loseButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>认输</string>
    </property>
   </widget>
   <widget class="QPushButton" name="head1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>110</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFike">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFike"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
